### PR TITLE
Layer Variation

### PR DESCRIPTION
**Pull request by OrionDevStar**
Finally github accept my terminal!
If you have any problems with the code, just send me a message.

### CODE REVIEW

**Comment by kevinramirez23** [comment]
hey, what does your additions do?

File: utils/regenerateMetadata.js [hunk 1]
```diff
@@ -1,9 +1,16 @@
 "use strict";
 
+<<<<<<< HEAD
```
Comment (MaciekBaron) [inline utils/regenerateMetadata.js:3]: You have merge conflict leftovers here I'm afraid

**Comment by tomvanleeuwennl** [comment]
If I read the code correctly it is possible to have a red top lid and a green bottom lid right? And the rarity is not functioning at these layers, or am I overlooking something?

Is it possible to have it like "if top lid is red, then bottom lid is also red". And is it possible to have the rarity still available at these labels?

**Comment by OrionDevStar** [comment]
S

**Pull request closed without merging**